src/docfx/build/xref/XrefResolver.cs

```diff
@@ -167,9 +176,22 @@ public XrefMapModel ToXrefMapModel(bool isLocalizedBuild)
                     })
                     .OrderBy(xref => xref.Uid)
                     .ToArray();
+
+                externalXrefs = _valdiateExternalXrefs.Value.GroupBy(uid => uid.Value).Select(uidGroup =>
+                {
+                    var docsetName = "";
```

Comment:
similar to `bool validateExternalXrefs`, you can also retrieve `docsetName` in `JsonSchemaTransformer`.

Reply:
OK